client/scss/components/_panel.scss

```diff
@@ -110,6 +106,8 @@ $header-button-size: theme('spacing.6');
 .w-panel__anchor--suffix {
   @include media-breakpoint-up(sm) {
     display: none;
+    width: $header-anchor-size;
```

Comment:
As per previous feedback, adding sizes here when we do not show it is not required. Please remove these width and height properties.